tests/coalaTest.py

```diff
@@ -322,3 +322,17 @@ def test_coala_without_color(self):
                 self.assertRegex(err, '^\[WARNING\]')
             self.assertEqual(
                 retval, 0, 'coala must return zero when there are no errors')
+
+    def test_coala_ignore_file(self):
+        with bear_test_module(), \
+                prepare_file(['#fixme'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                    coala.main, 'coala',
+                    '-c', os.devnull,
+                    '--non-interactive',
+                    '-f', re.escape(filename),
+                    '-i', re.escape(filename),
```

Comment:
Ah there were recent changes in our parsing on Windows: You can leave the `re.escape` calls, just `filename` is enough :)

Reply:
You need a test for the long `--ignore` flag as well

And as @Makman2 told: No `re.escape` of paths anymore. https://github.com/coala/coala/pull/4883 made that obsolete, and using `re.escape` for that purpose was anyway not a good idea ;)